hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -1,54 +1,52 @@
 import React from "react";
 import { render } from "@testing-library/react";
-import ItemTable from "./ItemTable";
+import {
+    PendingTable,
+    ReturnedTable,
+    CheckedOutTable,
+} from "components/dashboard/ItemTable/ItemTable";
 import { itemsP, itemsR, itemsC } from "pages/Dashboard/Dashboard";
```

Comment:
Eventually this is going to need to be moved somewhere else, eg into a mock data file once this is hooked up to redux. Doesn't have to be done now. I created a `testing` folder in my PR, so `testing/mockData.js` would probably be helpful.